bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,39 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
+
+ENVOY_COPTS = [
+    "-fno-omit-frame-pointer", "-Wall", "-Wextra", "-Werror", "-Wnon-virtual-dtor",
+    "-Woverloaded-virtual", "-Wold-style-cast", "-std=c++0x",
+    "-includesource/precompiled/precompiled.h", "-iquote", "include", "-iquote", "source"
```

Comment:
You mean in the includes attribute in envoy_cc_library? I tried that first time around, but since the target is relative to the package, it can't reference the build root. E.g. if the target being defined is in source/common/common, it can only specify includes in the cc_library that are a subdirectory of source/common/common.